templates/monitoring/kube_state_metrics_3scale_alerts.yaml

```diff
@@ -94,15 +94,24 @@ spec:
         for: 5m
         labels:
           severity: critical
-      - alert: ThreeScalePodCount
+      - alert: ThreeScaleZyncPodAvailability
         annotations:
+          message:  3Scale Zync has {{ "{{" }} printf "%.0f" $value {{ "}}" }} pods in a ready state. Expected at minimum of 2 pods.
           sop_url: https://github.com/RHCloudServices/integreatly-help/blob/master/sops/alerts_and_troubleshooting.md
-          message: Pod count for namespace {{ "{{" }} $labels.namespace {{ "}}" }} is {{ "{{" }} printf "%.0f" $value {{ "}}" }}. Expected at least 15 pods.
         expr: |
-          absent(sum(kube_pod_status_ready{condition="true",namespace="{{ index .Params "Namespace" }}"}) >= 14)
+          (1-absent(kube_pod_status_ready{condition="true",namespace="{{ index .Params "Namespace" }}"} * on (pod, namespace) kube_pod_labels{namespace="{{ index .Params "Namespace" }}", label_threescale_component="zync", label_threescale_component_element=""})) or count(kube_pod_status_ready{condition="true",namespace="{{ index .Params "Namespace" }}"} * on (pod, namespace) kube_pod_labels{namespace="{{ index .Params "Namespace" }}", label_threescale_component="zync", label_threescale_component_element=""}) != 2
         for: 5m
         labels:
-          severity: warning
+          severity: critical
+      - alert: ThreeScaleZyncDatabasePodAvailiblity
```

Comment:
Just noticed typo in `Availiblity`-> `Availability`

Reply:
:+1: